Gemfile

```diff
@@ -23,6 +23,8 @@ group :development, :test do
   gem 'pry-remote'
   gem 'rubycop'
   gem 'rspec'
+  gem 'rspec-rails', '~> 3.7'
```

Comment:
We need both 'rspec-rails' and 'rails-controller-testing' for testing controllers.